### ISSUE TITLE
[Accessbility]A11y_ Java Tooling for IntelliJ/Eclipse – New Feature_Link a Cluster-Link a Cluster_HD Insight_Keyboard: Focus on "X" (close) button is not visible

### ISSUE THREAD

**Issue by Amulya-95** [microsoft/azure-tools-for-java#3609]
[Check out Accessibility Insights!](https://github.com/microsoft/azure-tools-for-java/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHub Tags: #A11y_Java Tooling for IntelliJ/Eclipse–New Feature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #Java tooling for IntelliJ/Eclipse; #Keyboard; #MAS2.4.7

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.2
Build #IU-192.6603.28

**OS Details:**
OS: Windows (Version 1903, Build: 18362.356)
MAS Violated: MAS2.4.7

**Note: Similar issue is observed in** 
1.Link a Cluster-Link a Cluster_Livy Service
2.Create Spark Scala Project
3.Link SQL Server big data cluster

**Repro Steps:**
1)Launch IntelliJ IDEA (Darcula, High contrast, IntelliJ theme)
2)Tab till Create New Project button and press enter
3)Tab till view-->Tool Window-->Azure Explorer and press enter
4)Tab till signin-->copy and open & sign in to azure
5)Tab till select button and press enter
6)Tab till HDInsight and open context menu by using keyboard key
7)Tab till link a cluster and press enter
8)Verify whether focus is visible on 'X' (close) button or not

**Actual Result:**
Focus on "X" (close) button is not visible

**Expected Result:**
Focus on "X" (close) button should be visible

**User Impact:**
Keyboard users will find difficulty in navigating if keyboard focus is not visible on the controls

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

![Bug_LC HDInsight_Focus is not visible on Close button](https://user-images.githubusercontent.com/51435266/65497834-878d5700-ded8-11e9-911b-7efca03b6d56.png)

![Bug_Focus on 'X' button is not visible](https://user-images.githubusercontent.com/51435266/65487872-6f600c80-dec5-11e9-8ccb-7b0788d56759.png)

**Comment by t-rufang** [comment]
The first `Link A Cluster` dialog focus issue should be fixed after fixing #3616. For the second `New project` dialog, the close button on the dialog is owned by IntelliJ team and it's out of the scope of our toolkit.

**Comment by t-rufang** [comment]
- Importance of experience: Minimal. When focus is on “Close” button, for one tab focus is not visible.
- Quality of experience: Managable. When user hit the invisible focus, they just need to tab again and they will hit a visible focus button.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Alexandria-R** [comment]
Approved to close as external to intelliJ

**Comment by Amulya-95** [comment]
Closing this bug as per the discussion with SME

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749664
